exec/java-exec/src/main/java/org/apache/drill/exec/server/DrillbitContext.java

```diff
@@ -62,6 +63,7 @@
   private final CodeCompiler compiler;
   private final ScanResult classpathScan;
   private final LogicalPlanPersistence lpPersistence;
+  private final DrillOperatorTable table;
```

Comment:
Comment to explain that this is for static, built-in operators, not dynamic UDFs?